### ISSUE TITLE
Can't archive tasks

### ISSUE THREAD

**Issue by AnderRasoVazquez** [AnthonyDiGirolamo/todotxt-machine#22]
When I press `X` it doesn't archive tasks, it does nothing. This is what I have on my config file:

```
[settings]
file = ~/Documentos/Dropbox/todo/todo.txt
archive = ~/Documentos/Dropbox/todo/done.txt
```
I use version 2.0.5.

**Comment by AnthonyDiGirolamo** [comment]
Hey @AnderRasoVazquez I'm trying to reproduce this issue but not having any luck. I have the same config file setup and the todo.txt file has some completed items in it. When I hit `X` they are removed from my view and saved to the done.txt file. The only change to the todotxt-machine UI is that the view will be moved to the top and any completed todos will disappear. This works for me even if the done.txt does not exist yet. If it does exist the completed todos are appended to the file.

What is the contents of your todo.txt and done.txt files? Could those file paths be symlinks to locations that don't exist?

**Comment by AnderRasoVazquez** [comment]
They are simple txt files. I have deleted the done.txt file and it doesn't archive tasks or create a new one. Is it possible that there are some dependencies that I don't have?

**Comment by Querschlag** [comment]
I'm experiencing this bug as well.
Installed version 2.0.5 on Mac OS X 10.10 via pip.
Doesn't work either with or without a done.txt at the specified path in .todotxt-machinerc.

**Comment by AnderRasoVazquez** [comment]
I'm on Arch Linux if I install "todotxt-machine" package I can't archive tasks but with "todotxt-machine-git" yes.
todotxt-machine https://aur.archlinux.org/packages/todotxt-machine
todotxt-machine-git https://aur.archlinux.org/packages/todotxt-machine-git

**Comment by AnthonyDiGirolamo** [comment]
I didn't realize that it was on the AUR, thanks for letting me know!

**Comment by AnthonyDiGirolamo** [comment]
@Querschlag looks like 2.0.5 doesn't have the archiving feature yet. I'll push an update to pip soon, sorry for the confusion!

**Comment by AnthonyDiGirolamo** [comment]
pip installation method now includes archiving functionality. Fixed by commit 2699890
